setup.py

```diff
@@ -103,7 +103,7 @@
     python_requires=">=3.7,<3.11",
     install_requires=[
         "lark-parser~=0.12",
-        "fastapi~=0.65.2",
+        "fastapi~=0.65",
```

Comment:
In requirements.txt you list the dependency as fastapi==0.78.0
While here you list it as "fastapi~=0.65" 
That seems a bit strange to me.

Reply:
The `~=` operator installs the latest minor version compatible with the number, e.g.,

~=0.65 is equivalent to >=0.65,<1, whereas `~=0.65.2` is equivalent to `>=0.65.2<0.66` (I think).

This should allow people who are using FastAPI elsewhere in their stack to use the more relaxed version. The changes I made only effect schema generation which should be done with the pinned version anyway.

We could consider bumping this minimum requirement up at some point though.